docs/user/convert/parsing_and_validation.rst

```diff
@@ -0,0 +1,119 @@
+Parsing and Validation
+----------------------
+
+GeoMesa provides options for dealing with bad input records that can be configured on your converter. For example,
+you may want to skip over bad records for some input sources or fail entire files in other cases. The Converter
+framework provides a few options that can help control how errors in input data are handled.
+
+Validators
+~~~~~~~~~~
+
+At their core, Converters transform input streams into SimpleFeatures. Validators provide hooks to validate properties
+of those simple features before they are written to GeoMesa. For example, you may want to validate that there is a
+geometry field and that the geometry is valid.
+
+There are currently three validators available for usage with GeoMesa Converters:
+
+* has-geo - Ensure that the feature has a geometry and it is non-null
+* has-dtg - Ensure that the date has a geometry and it is non-null
+* z-index - Ensure the Geometry and Date are within the space/time bounds of GeoMesa Z-Index implementations
+(i.e. Z2, Z3, XZ2, XZ3)
+
+By default the has-geo and has-dtg validators are enabled. To enable other validators add this to the options block
```

Comment:
[picky style] I would use some sort of formatting (bold, italics, or quotation marks) for the names of the validators.